src/main/resources/view/MainWindow.fxml

```diff
@@ -45,7 +45,7 @@
                     </padding>
                 </StackPane>
 
-                <VBox fx:id="moduleList" styleClass="pane-with-border" minWidth="340" prefWidth="340"
+                <VBox fx:id="trackedModuleList" styleClass="pane-with-border" minWidth="340" prefWidth="340"
```

Comment:
Becareful what you change here. I don't see anywhere in the code that you change a variable of `moduleList` to `trackedModuleList`

Reply:
Noted. Changed back.